ROADMAP.md

```diff
@@ -56,10 +61,4 @@ V13 is targeted at developers with relayer and user experience improvements as w
 
 The **TokenFactory** will make developers' lives easier, and also make querying users' [DAO](https://daodao.zone/) tokens via MintScan and Keplr possible. By default, CosmWasm smart contracts accept native tokens. However, the only initial native tokens are the staking demons for most chains. This gives the ability for a user to create their token, and manage the tokenomics behind it. Then accept it just as they would any other denomination via the standard [x/bank](https://github.com/cosmos/cosmos-sdk/tree/main/x/bank) module.
 
-The **oracle** brings the ability to query external data sources, such as the price of JUNO or ATOM. This is a powerful tool for De-Fi applications on Juno like trading games, perpetual swaps, and more.
```

Comment:
No **oracle** anymore in the roadmap?

Reply:
This was the v13 section, we removed due to non-deterministic issues. We will be going with a wasm based oracle, which can be released at any time & will not require a software upgrade :)